doc/HACKING.md

```diff
@@ -124,6 +124,27 @@ Some data is stored as raw bytes, use `HEX(column)` to pretty print these.
 Make sure that clightning is not running when you query the database, as some
 queries may lock the database and cause crashes.
 
+#### Common variables
+Table `vars` contains global variables used by lightning node.
+
+    $ sqlite3 ~/.lightning/lightningd.sqlite3
+    SQLite version 3.21.0 2017-10-24 18:55:49
+    Enter ".help" for usage hints.
+    sqlite> .headers on
+    sqlite> select * from vars;
+    name|val
+    next_pay_index|2
+    bip32_max_index|4
+    ...
+
+Variables:
+* `next_pay_index` last resolved invoice counter.
```

Comment:
Nit: Actually as by the name, `next_pay_index` is the next resolved invoice counter that will get assigned.